simplq/src/components/pages/Status/ContactAdmin.jsx

```diff
@@ -1,16 +1,16 @@
 import React from 'react';
```

Comment:
Just delete all files above this comment in the PR... and manually revert any change that you didn't make, that's the easiest way out.

Reply:
I just created a new branch and manually pushed my fixes there. Was easier than trying to fix this mess. Sorry for that :-(